libraries/botbuilder/src/botFrameworkAdapter.ts

```diff
@@ -1156,27 +1169,28 @@ export class BotFrameworkAdapter extends BotAdapter implements IUserTokenProvide
         return serviceUrl && !serviceUrl.toLowerCase().startsWith('http');
      }
 
-    private async authenticateConnection(req: WebRequest, appId?: string, appPassword?: string, channelService?: string): Promise<boolean> {
-        if (!appId || !appPassword) {
+    private async authenticateConnection(req: WebRequest, channelService?: string): Promise<void> {
+        if (!this.credentials.appId || !this.credentials.appPassword) {
```

Comment:
same here on password

Reply:
I like that pattern better, it's been bothering me that we make the GetToken() call here and don't do anything with it.

Reply:
I've removed the falsey password check, and taken the `getToken()` logic out of this method.